numpy/core/numeric.py

```diff
@@ -1436,6 +1436,10 @@ def rollaxis(a, axis, start=0):
     """
     Roll the specified axis backwards, until it lies in a given position.
 
+    This function continues to be supported for backward compatibility, but you
+    should prefer np.moveaxis. The np.moveaxis function was added in NumPy
+    1.11.
```

Comment:
Both, I think. There's nothing special about "np" - remember, the module's called "numpy" (but that's implied anyway in references)